.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: dotnet build
```

Comment:
I think by default this will do a debug build?  I think that is ok, since this will just create versions for testing.  Is that your thinking here too?

Reply:
the dotnet publish below will for sure do a release build and that's what is used for the upload. I'm not even sure if this build is needed as dotnet test implicitly builds i think.